src/main/scala/com/fulcrumgenomics/vcf/AssessPhasing.scala

```diff
@@ -642,6 +639,7 @@ private[vcf] object PhaseCigar {
       case _ => true
     }.flatMap { case (t, c) => // collect metrics but only keep sites where either variant context (i.e. truth or call) is phased.
       val ctxs                  = Seq(t, c).flatten // NB: can be 1 or 2 contexts here
+      @unused // TODO: should this be unused or should it be used to set something upon the metric?
```

Comment:
There is a variable here that _**looks**_ like it should be used to update the metric output of `AssessPhasing` but I'm uncertain about the expected implementation due to my unfamiliarity with this tool.

Reply:
https://github.com/fulcrumgenomics/fgbio/issues/1048